.github/workflows/merge-main.yml

```diff
@@ -64,13 +62,27 @@ jobs:
           overwrite: ${{ matrix.overwrite }}
           penetration_test: false
           parameters:
-            -p PROMOTE=${{ github.repository }}/${{ matrix.name }}:${{ env.ZONE }}
             -p ZONE=${{ env.ZONE }} ${{ matrix.parameters }}
           post_rollout: ${{ matrix.post_rollout }}
 
+  prod-promotions:
+    name: Promote images to PROD
+    needs: [deploy-test]
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        component: [api, admin, db, public]
+    steps:
+      - uses: shrink/actions-docker-registry-tag@v4
+        with:
+          registry: ghcr.io
+          repository: ${{ github.repository }}/${{ matrix.component }}
```

Comment:
I notice this is different from the PROMOTE value in merge-main which uses matric name + env.zone, not sure why that would be.

Reply:
We've moved away from promote vars in favour of PR numbers all the way through. The plan is to bring FOM all the way up to date. ... eventually!